app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str, skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf blacksmith {self.nickname} "
+                f"with skill of the {self._skill_level} level")
+
+    def get_rating(self) -> int:
+        return self._skill_level
+
+    def eat_favourite_dish(self) -> None:
+        super().eat_favourite_dish()

```

Comment:
The `eat_favourite_dish` method calls `super().eat_favourite_dish()`, which is an abstract method in the `Dwarf` class. Ensure that the `Dwarf` class's `eat_favourite_dish` method is not abstract or has a proper implementation. Otherwise, this call will lead to a runtime error.